package.json

```diff
@@ -56,12 +56,12 @@
   "dependencies": {
     "@cucumber/gherkin": "^29.0.0",
     "@wdio/logger": "^9.0.4",
-    "@wdio/reporter": "^8.15.10",
+    "@wdio/reporter": "^9.0.0",
     "fs-extra": "^11.1.1",
     "strip-ansi": "^7.1.0"
   },
   "peerDependencies": {
-    "@wdio/cucumber-framework": "^8.0.0",
+    "@wdio/cucumber-framework": "^9.0.0",
     "@wdio/globals": "^8.0.0",
     "@wdio/types": "^8.0.0",
     "webdriverio": "^8.0.0"
```

Comment:
First of all, thank you Hanna for the PR and work 👌 .

One little question:
Why did you not update the other wdio peerDependencies to v9?

Reply:
Thanks for pointing this out, updated